src/Raven.Server/Smuggler/Documents/DatabaseSource.cs

```diff
@@ -125,6 +126,15 @@ public Task<DatabaseItemType> GetNextTypeAsync()
             return Task.FromResult(_types[_currentTypeIndex++]);
         }
 
+        public Task<DatabaseRecord> GetShardedDatabaseRecordAsync()
+        {
+            using (_database.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            using (var rawRecord = _database.ServerStore.Cluster.ReadRawDatabaseRecord(context, ShardHelper.ToDatabaseName(_database.Name)))
```

Comment:
Can just call `ReadDatabaseRecord`

Reply:
with ReadDatabaseRecord we get shard database record and not real database record